code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -9,6 +9,13 @@
 
 /mob/living/carbon/human/movement_delay()
 	. = ..()
+	if(CHECK_MOBILITY(src, MOBILITY_STAND) && m_intent == MOVE_INTENT_RUN && IS_SPRINTING(src))
```

Comment:
One day these should be converted into movespeed modifiers, just so you know.

Reply:
yep i am just delaying the inevitable out of laziness :weary: